Unable to import `@formkit/themes/tailwindcss` with TypeScript

### Reproduction

https://codesandbox.io/s/goofy-currying-rnursq?file=/tailwind.config.ts

### Describe the bug

![image](https://user-images.githubusercontent.com/203980/188419778-e8cf1a1f-d97d-4199-8165-7d6fbe04fdeb.png)

__Error message:__
```Cannot find module '@formkit/themes/tailwindcss' or its corresponding type declarations.ts(2307)```
### Environment

• OS: not related
• Browser: not related
• Version: ^1.0.0-beta.11-0dd967c

--- a/packages/themes/package.json
+++ b/packages/themes/package.json
@@ -29,6 +29,19 @@
       "types": "./dist/index.d.ts"
     }
   },
+  "typesVersions": {
+    "*": {
+      "windicss": [
+        "./dist/windicss/index.d.ts"
+      ],
+      "tailwindcss": [
+        "./dist/tailwindcss/index.d.ts"
+      ],
+      "unocss": [
+        "./dist/unocss/index.d.ts"
+      ]
+    }
+  },
   "keywords": [
     "vue",
     "forms",

